Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG deps="findutils hostname jq libpq openssl procps-ng ruby shared-mime-info tzdata"
-ARG devDeps="gcc gcc-c++ gzip libffi-devel make openssl-devel patch postgresql postgresql-devel redhat-rpm-config ruby-devel tar util-linux xz"
+ARG devDeps="gcc gcc-c++ gzip libffi-devel make openssl-devel patch postgresql-devel redhat-rpm-config ruby-devel tar util-linux xz"
 ARG extras=""
 ARG prod="true"
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS build
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS build
```

Comment:
This can remain UBI 9.  Only thing that needs to be ubi8 is the final result that gets uploaded to quay.